code/modules/hydroponics/grown.dm

```diff
@@ -1382,3 +1382,20 @@ obj/item/weapon/reagent_containers/food/snacks/grown/shell/eggy/add_juice()
 		reagents.add_reagent("fartium", 1 + round((potency / 10), 1))
 		bitesize = 1 + round(reagents.total_volume / 2, 1)
 
+/obj/item/weapon/reagent_containers/food/snacks/grown/limb_spawn
+	seed = /obj/item/seeds/limbseed
+	name = "limbplant"
+	desc = "A cluster of limbs sprouting from a stem."
+	icon_state = "limbplant"
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/limb_spawn/canconsume(mob/eater, mob/user)
+	return 0
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/limb_spawn/attack_self(mob/user as mob)
+	if(user)
+		user.unEquip(src)
+	var/obj/item/organ/limb/L = newBodyPart(pick("r_arm", "l_arm", "r_leg", "l_leg"))
```

Comment:
What if instead of this picking a part a random it would choose the part you targetted?